source/reference/options.txt

```diff
@@ -468,6 +474,52 @@ Authentication Options
    - ``none``, the effect is the same as setting
      ``authMechanismProperties=CANONICALIZE_HOST_NAME:false``.
 
+.. option:: --oidcFlows
+
+   Specifies OpenID Connect flows in a comma-separated list. 
+   The OpenID connect flows specify how ``mongosh``interacts with the identity 
+   provider for the authentication process. ``mongosh`` supports the following 
+   OpenID Connect flows:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 25 75
+
+      * - OpenID Connect Flow
+        - Description
+
+      * - ``auth-code``
+        - Default. ``mongosh`` redirects you to the identity provider log-in screen.
+
+      * - ``device-auth``
+        - ``mongosh`` provides you with a URL and code to finish authentication.
+          This is considered a less secure OpenID Connect flow.
+
+   To set ``device-auth`` as a fallback option to ``auth-code``, see the following 
+   example:
+
+   .. code-block:: bash 
+
+      mongosh 'mongodb://localhost/' --authenticationMechanism MONGODB-OIDC --oidcFlows=auth-code,device-auth
+
+.. option:: --oidcRedirectUri
+
+   Specifies a URI where the identity provider redirects you after authentication.
+   The URI must match the configuration of the identity provider used by the server.
+   The default is ``http://localhost:27097/redirect``.
+
+.. option:: --oidcTrustedEndpoint
+
+   Specifies a connection to a trusted endpoint that is not Atlas or localhost. 
+   Only connect to hosts that you know are safe. 
```

Comment:
```suggestion
   Only use this option when connecting to servers that you trust.
```

(This is the wording from the Compass docs, I think it also makes sense to align that here)

Reply:
Done.